js/__internal/grids/grid_core/columns_controller/module.ts

```diff
@@ -363,11 +363,14 @@ export class ColumnsController extends modules.Controller {
 
   getCommonSettings(column?) {
     const commonColumnSettings = (!column || !column.type) && this.option('commonColumnSettings') || {};
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
     const groupingOptions: any = this.option('grouping') || {};
```

Comment:
Question.

Can the 'grouping' or 'groupPanel' options value be false but not null or undefined?
What is this scenario?

Reply:
Can't
Disabled instead of fixing only because it was done automatically, I'll fix it